src/views/bouquets/BouquetEditView.vue

```diff
@@ -69,49 +181,84 @@ onMounted(() => {
-          />
-          <DsfrAlert v-if="errorMessage" type="warning" :title="errorMessage" />
-        </div>
+                <DsfrInput
+                  v-model="form.name"
+                  class="fr-mt-1w fr-mb-4w"
+                  type="text"
+                  placeholder="Mon bouquet"
+                  :label-visible="true"
+                  label="Sujet du bouquet"
+                />
 
-        <form @submit.prevent="onSubmit()">
-          <DsfrInput
-            v-model="form.name"
-            class="fr-mt-1w fr-mb-4w"
-            type="text"
-            placeholder="Mon bouquet"
-            :label-visible="true"
-            label="Sujet du bouquet"
-          />
-
-          <Tooltip
-            title="Objectif du bouquet"
-            name="tooltip__objectif"
-            text="Ajoutez ici l'ensemble des informations nécessaires à la compréhension, l'objectif et l'utilisation du bouquet. N'hésitez pas à indiquer la réglementation ou une documentation liée au bouquet."
-          />
-          <Tooltip
-            title="Utilisez du markdown pour mettre en forme votre texte"
-            name="tooltip__markdown"
-            text="* simple astérisque pour italique *<br/> ** double astérisque pour gras **<br/> # un dièse pour titre 1<br/> ## deux dièses pour titre 2<br/> *  astérisque pour une liste<br/> lien : [[https://exemple.fr]]"
-          />
-          <DsfrInput
-            v-model="form.description"
-            class="fr-mt-1w fr-mb-2w"
-            placeholder="Ma description"
-            :label-visible="true"
-            :is-textarea="true"
-          />
-
-          <DsfrButton class="fr-mt-4w" type="submit" label="Suivant" />
-        </form>
+                <Tooltip
+                  title="Objectif du bouquet"
+                  name="tooltip__objectif"
+                  text="Ajoutez ici l'ensemble des informations nécessaires à la compréhension, l'objectif et l'utilisation du bouquet. N'hésitez pas à indiquer la réglementation ou une documentation liée au bouquet."
+                />
+                <Tooltip
+                  title="Utilisez du markdown pour mettre en forme votre texte"
+                  name="tooltip__markdown"
+                  text="* simple astérisque pour italique *<br/> ** double astérisque pour gras **<br/> # un dièse pour titre 1<br/> ## deux dièses pour titre 2<br/> *  astérisque pour une liste<br/> lien : [[https://exemple.fr]]"
+                />
+                <DsfrInput
+                  v-model="form.description"
+                  class="fr-mt-1w"
+                  placeholder="Ma description"
+                  :label-visible="true"
+                  :is-textarea="true"
+                />
+            </div>
+          </div>
+        </div>
+        <DsfrButton type="button" class="fr-mt-2w" label="Suivant" @click.prevent="validateAndMoveToStep2(2)" />
       </div>
-    </div>
+    
+      <div v-show="currentStep === 2">
```

Comment:
I would put the steps in different components, it will be hellish when more steps come in, and the diff is already polluted with step 1 changes.

Reply:
+1

Reply:
I've already thought of doing this, but I was getting an error because form.name and form.description passed to the step2 component were returning errors. 
I've already discussed this with @bonjourmauko , so I'll leave it as it is for now and come back to it in another iteration to continue moving forward quickly and efficiently on the rest of the elements to be developed.

Reply:
@abulte Il y a un _spike_ sur cela by @edelagnier ici https://github.com/ecolabdata/ecospheres-front/pull/145 . Tes retours sont très bienvenus :)